.github/workflows/ocwm-pre-meeting-check.yml

```
@@ -0,0 +1,83 @@
+name: OCWM Pre-Meeting Check
+
+on:
+  schedule:
+    - cron: '50 21 * * 1'  # Runs at 21:50 UTC, which is 10 minutes before 14:00 PT every Monday
```

```suggestion
    - cron: '50 18 15-21 * 1'  # Runs at 11:50 AM PT on the 3rd Monday of the month
```
I am suggesting this change because we just decided to make the meetings mothly the 3rd monday of the month and 2 hours earlier as 12:00 PT